index.html

```diff
@@ -8,6 +8,37 @@
     <title>Personal Portfolio</title>
 </head>
 <body>
-    
+    <nav class="toolbar">
+        <figure class="logo-wrapper">
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
```

Comment:
**REQUIRED:** The logo should be `a` element, please try to put the logo image inside an `a` element, so whenever user clicks on logo in the future he/she will go back to the top of portfolio.